graphql_compiler/tests/test_backend.py

```diff
@@ -0,0 +1,8 @@
+# Copyright 2018-present Kensho Technologies, LLC.
+class TestBackend(object):
```

Comment:
Do we need a class here? Can these just be module constants instead, and we can refer to them via the module name like `test_backend.POSTGRES`?

Reply:
Yeah that sounds good.